infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/K8sContainerResolver.java

```diff
@@ -58,6 +63,7 @@ public Container resolve() throws InfrastructureException {
     Container container =
         new ContainerBuilder()
             .withImage(cheContainer.getImage())
+            .withImagePullPolicy(imagePullPolicy)
```

Comment:
Can you elaborate on how k8s would react in case image pull policy is set to null?

Reply:
That is described on property description: will be default behavior (always on latest tag and ifnotpresent otherwise)

Reply:
ok, thanks